app/views/guides/show.html.erb

```diff
@@ -1,94 +1,203 @@
-<%= @guide.user.age %>,
-<%= @guide.location %>
-<br>
-<legend>Availability</legend>
-<% if @guide.availability[0] %>
-  <b>Sunday</b>
-  <br>
-<% end %>
 
-<% if @guide.availability[1] %>
-  <b>Monday</b>
-  <br>
-<% end %>
-
-<% if @guide.availability[2] %>
-  <b>Tuesday</b>
-  <br>
-<% end %>
-
-<% if @guide.availability[3] %>
-  <b>Wednesday</b>
-  <br>
-<% end %>
-
-<% if @guide.availability[4] %>
-  <b>Thursday</b>
-  <br>
-<% end %>
-
-<% if @guide.availability[5] %>
-  <b>Friday</b>
-  <br>
-<% end %>
-
-<% if @guide.availability[6] %>
-  <b>Saturday</b>
-  <br>
-<% end %>
-
-<br>
-
-<!-- Rating Star display code starts here -->
-
-<strong>Overall Rating: </strong>
-<% if @guide.reviews.any? %>
-  <% average = @guide.reviews.average(:rating) %>
-<% else %>
-  <% average = 0 %>
-<% end %>
-
-<% parts = average.divmod(1) %>
-<% stars = parts[0] %>
-
-<% if parts[1] >= 0.5 %>
-  <% halfStar = true %>
-  <% blankStars = 4 - stars %>
-<% else %>
-  <% halfStar = false %>
-  <% blankStars = 5 - stars %>
-<% end %>
-
-<% stars.to_i.times do  %>
-  <!-- Uncomment below when Bootstrap is moved from Front End -->
-  <!-- <span class="fa fa-star"></span> -->
-  <span><b>FS</b></span>
-<% end %>
-
-<% if halfStar %>
-  <!-- Uncomment below when Bootstrap is moved from Front End -->
-  <!-- <span class="fa fa-star-half-o"></span> -->
-  <span><b>HS</b></span>
-<% end %>
-
-<% blankStars.to_i.times do %>
-  <!-- Uncomment below when Bootstrap is moved from Front End -->
-  <!-- <span class="fa fa-star-o"></span> -->
-  <span><b>BS</b></span>
-<% end %>
-
-<!-- Rating Star display ends here -->
-
-  <div class="col-md-8">
-  <% if @guide.reviews.any? %>
-    <h3>Reviews (<%= @guide.reviews.count %>)</h3>
-    <ol class="reviews">
-      <%= render @reviews %>
-    </ol>
-    <%= will_paginate @reviews %>
-  <% end %>
-</div>
+<body>
+  <!-- Page Content -->
+  <div class="container">
+
+    <!-- Page Heading -->
+    <div class="row">
+        <div class="col-lg-12">
+            <h1 class="page-header">
+                Guide Profile
+            </h1>
+        </div>
+    </div>
+    <!-- /.row -->
+
+    <!-- Guide Profile Section -->
+    <div class="row">
+      <!-- Main Guide Info: Name, Location, Sport, Pricing, Ratings -->
+      <div class="col-lg-12">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <div class="col-xs-12 col-sm-5">
+              <img class="img img-circle img-responsive full-width"
+                src="img/guide-n-location/guide1.jpg"
+                style="position:relative; overflow: hidden;" alt="">
+            </div>
+            <div class="col-sm-offset-1 col-xs-12 col-sm-6">
+              <h1><strong><%= @guide.user.name %></strong></h1>
+              <h3><strong>Location: </strong><%= @guide.location %></h3>
+              <h3><strong>Sports: </strong><%= @guide.specialty %></h3>
+              <h3><strong>Pricing: </strong>$<%= @guide.rate %>/hour</h3>
```

Comment:
<h3><strong>Pricing: </strong><%= number_to_currency(guides.rate) %>/hour</h3>
will format the rate to 2 decimal pts and add the $ automatically.